### PR TITLE
Add test for no lock pot distribution

### LINKED ISSUE #186: Test what happens when you distribute to a pot with no locks
Need to make sure this doesn't panic or otherwise break things.  What happens to the coins in that case?

### PR DESCRIPTION
**Pull request by antstalepresh**
closes #186

### CODE REVIEW

File: x/incentives/keeper/pot_test.go [hunk 1]
```diff
@@ -331,3 +331,48 @@ func (suite *KeeperTestSuite) TestPerpetualPotOperations() {
 	rewardsEst = suite.app.IncentivesKeeper.GetRewardsEst(suite.ctx, lockOwner, []lockuptypes.PeriodLock{}, 100)
 	suite.Require().Equal(sdk.Coins(nil), rewardsEst)
 }
+
+func (suite *KeeperTestSuite) TestNoLockPotDistribution() {
```
Comment (ValarDragon) [inline x/incentives/keeper/pot_test.go:335]: Should we have a test for the non-perpetual variant as well?

The test for the perpetual variant LGTM

Comment (antstalepresh) [reply x/incentives/keeper/pot_test.go]: Added test for non perpetual pot distribution for no locks

File: x/incentives/keeper/pot_test.go [hunk 2]
```diff
@@ -376,3 +376,48 @@ func (suite *KeeperTestSuite) TestNoLockPotDistribution() {
...
+	suite.Require().Equal(pots[0].String(), expectedPot.String())
+
+	// start distribution
+	suite.ctx = suite.ctx.WithBlockTime(startTime)
+	pot, err := suite.app.IncentivesKeeper.GetPotByID(suite.ctx, potID)
+	suite.Require().NoError(err)
+	err = suite.app.IncentivesKeeper.BeginDistribution(suite.ctx, *pot)
+	suite.Require().NoError(err)
+
+	// distribute coins to stakers, since it's perpetual distribute everything on single distribution
```
Comment (ValarDragon) [inline x/incentives/keeper/pot_test.go:414]: stakers should be changed with bonded LPs

File: x/incentives/keeper/pot_test.go [hunk 3]
```diff
@@ -376,3 +376,48 @@ func (suite *KeeperTestSuite) TestNoLockPotDistribution() {
...
+	distrCoins, err := suite.app.IncentivesKeeper.Distribute(suite.ctx, *pot)
+	suite.Require().NoError(err)
+	suite.Require().Equal(distrCoins, sdk.Coins(nil))
+
+	// check state is same after distribution
+	pots = suite.app.IncentivesKeeper.GetNotFinishedPots(suite.ctx)
+	suite.Require().Len(pots, 1)
+	suite.Require().Equal(pots[0].String(), expectedPot.String())
```
Comment (ValarDragon) [inline x/incentives/keeper/pot_test.go:422]: Why doesn't filled epochs increment?

Comment (antstalepresh) [reply x/incentives/keeper/pot_test.go:422]: If no distribution, no increment

File: x/incentives/keeper/pot_test.go (hunk 2 above)
Comment (antstalepresh) [reply x/incentives/keeper/pot_test.go:414]: I don't get the meaning of "stakers should be changed with bonded LPs"

Comment (ValarDragon) [reply x/incentives/keeper/pot_test.go:414]: just the word in the comment says "stakers", I believe that it should read "bonded LPs"

File: x/incentives/keeper/pot_test.go (hunk 3 above)
Comment (ValarDragon) [reply x/incentives/keeper/pot_test.go:422]: Oh interesting, lets make a note to add this to the spec. I like this design =)

Comment (antstalepresh) [reply x/incentives/keeper/pot_test.go:422]: Ok

File: x/incentives/keeper/pot_test.go (hunk 2 above)
Comment (antstalepresh) [reply x/incentives/keeper/pot_test.go:414]: I see

**Pull request merged**